core/unit_test/Makefile

```diff
@@ -289,6 +289,7 @@ ifeq ($(KOKKOS_INTERNAL_USE_HIP), 1)
 	OBJ_HIP += TestHIP_Reductions.o
 	OBJ_HIP += TestHIP_MDRange_a.o TestHIP_MDRange_b.o TestHIP_MDRange_c.o TestHIP_MDRange_d.o TestHIP_MDRange_e.o
 	OBJ_HIP += TestHIP_Spaces.o
+	OBJ_HIP += TestHIP_Memory_Requirements.o
```

Comment:
That makes me realize how we definitely have not been good about adding new tests to the generated GNU makefiles lately...

Reply:
Maybe we should bring this up at the weekly meeting?